retrievalmarket/impl/common.go

```diff
@@ -7,7 +7,7 @@ import (
 	"github.com/ipld/go-ipld-prime/traversal/selector/builder"
 )
 
-func allSelector() ipld.Node {
+func AllSelector() ipld.Node {
```

Comment:
Seems like while you are exporting this, this is code snippet is used all over the place in tests and in the storage market...